### PR TITLE
Changing travis.yml to use mongo 3

### PR DESCRIPTION
**Pull request by eduardomoroni**

### CODE REVIEW

**Review by rafaelsmag — APPROVED** [review]
Aí sim.. Hehe

**Comment by eduardomoroni** [comment]
@Aprendfy/dev-team Up and Running

File: .travis.yml [hunk 1]
```diff
@@ -4,10 +4,17 @@ node_js:
 cache:
   directories:
     "node_modules"
-services: 
-  - mongodb
+sudo: false
```
Comment (eduardomoroni) [inline .travis.yml:7]: https://github.com/travis-ci/travis-ci/issues/3694#issuecomment-155186560

Comment (hebertporto) [reply .travis.yml:7]: sucesso.

**Pull request merged**